railties/lib/rails/tasks/engine.rake

```diff
@@ -62,7 +62,7 @@ namespace :db do
   desc "Load the seed data from db/seeds.rb"
   app_task "seed"
 
-  desc "Create the database, load the schema, and initialize with the seed data (use db:reset to also drop the db first)"
+  desc "Create the database, load the schema, and initialize with the seed data (use db:reset to also drop the DB first)"
```

Comment:
let's change to database.

Reply:
@rafaelfranca thanks! you always give the things a better twist ❤️ 

I have changed it as suggested and also updated the PR title and description.